extensions/blocks/contact-form/components/jetpack-crm-connection-settings.js

```diff
@@ -64,19 +69,18 @@ function CRMConnectionSettings( props ) {
 				);
 			}
 
-			if ( pluginState.ACTIVE === jetpackCRMPlugin ) {
-				if ( semver.satisfies( jetpackCRMVersion, '3.0.19 - 4.0.0' ) ) {
-					return (
-						<p className="jetpack-contact-form__crm_text">
-							{ __(
-								'Contacts from this form will be stored in Jetpack CRM if the CRM Jetpack Forms extension is active.',
-								'jetpack'
-							) }
-						</p>
-					);
-				}
-
-				return null;
+			if (
+				pluginState.ACTIVE === jetpackCRMPlugin &&
+				semver.satisfies( jetpackCRMVersion, '3.0.19 - 4.0.0' )
```

Comment:
We need to return null when the CRM plugin is active and version is >4.0. We no longer display any text in that scenario, just the toggle. I'll push a commit that addresses this scenario.

Reply:
See commit 5e433a0.